pkgs/by-name/ki/kikit/default.nix

```
@@ -25,16 +25,16 @@ let
 in
 buildPythonApplication rec {
   pname = "kikit";
-  version = "1.7.0";
-  format = "setuptools";
+  version = "1.7.1";
+  pyproject = true;
 
   disabled = pythonOlder "3.7";
 
   src = fetchFromGitHub {
     owner = "yaqwsx";
     repo = "KiKit";
     tag = "v${version}";
-    hash = "sha256-b4I+RvxjCAcHam/uhvUQZrnET9QWeBlhobLeXkMcdRA=";
+    hash = "sha256-GG0OXPoTy219QefQ7GwMen4u66lPob5DI8lU9sqwaRQ=";
   };
 
   propagatedBuildInputs = [
```

```suggestion
  dependencies = [
```
this should also be updated when using pyproject = true

also we probably need to add build-system

---

Switched `propagatedBuildInputs` -> `dependencies`.

> also we probably need to add build-system

Done. It's disappointing that the package builds even if I forget to specify that :(